### PR TITLE
tutorial-02: updates response examples

### LINKED ISSUE #50: Tutorials review
Review all tutorials to ensure they work as intended with Invenio v3.2

### PR DESCRIPTION
**Pull request by roksys**
* Closes #50

### CODE REVIEW

File: 07-data-models-new-field/README.md [hunk 1]
```diff
@@ -125,10 +125,10 @@ After executing the command you should see in your console the following output:
...
 ```
 
-It seems that our request wasn't successfull. By checking again the error message we can see that in our request
+It seems that our request wasn't successful. By checking again the error message we can see that in our request
 the `owner` field has a `string` value rather than an `integer`. But who validated that?
 
-If you remember we talked earlier about `loaders` and specifically we updated our `marshmallow` schema. But how are these related? To answer that let's talk about what is the responsibility of the `loaders`. It's purpose is to load the data which is passed when doing a request to create a new record, validate them by using our `marshmallow` schema and transform them in our internal representation.
+If you remember we talked earlier about `loaders` and specifically we updated our `marshmallow` schema. But how is it related? To answer that let's talk about what is the responsibility of the `loaders`. It's purpose is to load the data which is passed by a request to create a new record, validate it by using our `marshmallow` schema and transform it into our internal representation.
```
Comment (ppanero) [inline 07-data-models-new-field/README.md]:
```suggestion
If you remember we talked earlier about `loaders` and specifically we updated our `marshmallow` schema. But how is it related? To answer that let's talk about what is the responsibility of the `loaders`. Its purpose is to load the data which is passed from the request to create a new record, validate it using our `marshmallow` schema and transform it into our internal representation.
```

File: 07-data-models-new-field/README.md [hunk 2]
```diff
@@ -145,23 +145,26 @@ Now you should see an output similar to the below:
...
+  "_bucket": "60f2b083-8f7b-4aba-a00e-09e3bb3e12af",
+  "created": "2019-11-25T07:41:44.620275+00:00",
+  "id": "1",
   "links": {
-    "self": "https://localhost:5000/api/records/2"
+    "files": "https://localhost:5000/api/records/1/files",
+    "self": "https://localhost:5000/api/records/1"
   },
   "metadata": {
+    "$schema": "https://my-site.com/schemas/records/record-v1.0.0.json",
```
Comment (ppanero) [inline 07-data-models-new-field/README.md]: moderate: Is not the fault of the work, maybe Invenio is not fully working. Is this supposed to be in the output?

**Review by ppanero — APPROVED** [review]

File: 07-data-models-new-field/README.md (hunk 1 above)
Comment (diegodelemos) [reply 07-data-models-new-field/README.md]: minor:
```diff
- Its purpose is to load the data which is passed from the request to create a new record ...
+ Its purpose is to load the data received in a request to create a new record ...
```

**Review by diegodelemos — APPROVED** [review]

File: 07-data-models-new-field/README.md (hunk 2 above)
Comment (topless) [reply 07-data-models-new-field/README.md]: This was changed, in my pr #71

**Pull request merged**